extensions/wikia/ArticleVideo/scripts/jwplayer-autoplay-toggle.js

```diff
@@ -0,0 +1,35 @@
+define('wikia.articleVideo.jwPlayerAutoplayToggle', ['wikia.geo', 'wikia.instantGlobals', 'wikia.cookies'], function (geo, instantGlobals, cookies) {
+
+	return function (playerInstance) {
+		var inAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries),
+			// TODO extract it to separate module
+			autoplayCookieName = 'featuredVideoAutoplay';
+
+		if (inAutoplayCountries) {
+			function addAutoplayToggleButton() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Do not use function declarations within blocks. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionDeclarationsWithinBlocks)